nbextensions/upload_it/main.js

```diff
@@ -79,14 +79,26 @@ define([
             window.console.log("Uploading ", notebook.notebook_path, " to ", url, formdata);
             $.ajax({
               url: url,
+              xhrFields: {withCredentials: true},
               data: formdata,
               contentType: false,
               processData: false,
               method: "POST",
               success: function(data, status, jqXHR) {
-                window.console.log("Upload OK", data);
+                // Open the report in a new tab.
+                var u = new URL(url);
```

Comment:
style: Let's use `const`.
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Thanks for the pointer. Done.